core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/dr/BackupRestoreSaltConfigGenerator.java

```diff
@@ -43,7 +56,17 @@ public SaltConfig createSaltConfig(String location, String backupId, String rang
         disasterRecoveryValues.put(OBJECT_STORAGE_URL_KEY, fullLocation);
         disasterRecoveryValues.put(RANGER_ADMIN_GROUP_KEY, rangerAdminGroup);
         disasterRecoveryValues.put(CLOSE_CONNECTIONS, String.valueOf(closeConnections));
-
+        if (!skipDatabaseNames.isEmpty()) {
```

Comment:
this seems like a perfect place to read the skipped databases from `application.yml` file

Reply:
You're right that the value is going to be the same, but it's not going to be applied to every backup